packages/shared-ux/button_toolbar/src/popover/popover.tsx

```diff
@@ -29,7 +29,14 @@ export type Props = AllowedButtonProps &
 /**
  * A button which opens a popover of additional actions within the toolbar.
  */
-export const ToolbarPopover = ({ type, label, iconType, children, ...popover }: Props) => {
+export const ToolbarPopover = ({
+  type,
+  label,
+  iconType,
+  size = 'm',
```

Comment:
Nice call to have the size here default to `medium`. That way we don't have to change Canvas at all.